src/CreatePoserFactory.php

```diff
@@ -121,21 +114,20 @@ protected function createFactory($factoryName, $className = null)
 
         $this->line("");
         $this->info("Please consider starring the repo at https://github.com/lukeraymonddowning/poser");
+        return 0;
     }
 
     protected function createAllFactories()
     {
+        $namespace = trim(modelsNamespace(), '\\');
+        $models = ClassFinder::getClassesInNamespace($namespace);
+        if (!\is_array($models) || empty($models)) {
```

Comment:
As far as I can see, this will always return an array, even if it is empty. So the first check is redundant, no?

Reply:
I _did_ get a case where is was `false`, not sure how to replicate getting it to return false but nonetheless I thought it best to leave in the check anyway. Can remove if you'd prefer

Reply:
If it returns `false`, the second clause will pass anyway, so we might as well strip it out.